spec/gameSpec.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+describe('Game', function() {
+  var game;
+  beforeEach(function() {
+    game = new Game();
+  });
+  it('has a score of 0', function() {
+    expect(game.score).toEqual(0)
+  });
+  it('keeps score of the game', function() {
+    game.roll(4)
+    expect(game.score).toEqual(4)
+  });
+  it('adds multiple rolls together', function() {
+    game.roll(4)
+    game.roll(3)
+    game.roll(6)
+    game.roll(2)
+    expect(game.score).toEqual(15)
+  });
+  it('keeps track of the frame', function() {
+    game.roll(4)
+    game.roll(3)
+    game.roll(6)
+    expect(game.currentFrame()).toEqual(2)
+  });
+  it('limits to 10 frames', function() {
+    for ( var i = 0; i < 20; i++) {
+      game.roll(0)
+    }
+    expect(function() { game.roll(0) }).toThrow('Game over')
+  });
+  it('ends the frame if a strike is bowled', function() {
+    game.roll(10)
+    expect(game.currentFrame()).toEqual(2)
+  });
+  it('calls the final frame for the end of the game', function() {
+    game.roll(4,4)
+    game.roll(4,4)
+    game.roll(4,4)
+    game.roll(4,4)
+    game.roll(4,4)
+    game.roll(4,4)
+    game.roll(4,4)
+    game.roll(4,4)
+    game.roll(4,4)
+    game.roll(4)
```

Comment:
you could have used a for loop for these lines

Reply:
Either a for loop or you can define a function at the bottom of the tests!

Reply:
I totally forgot I hadn't refactored that before committing! Looks dreadful, doesn't it...